MacaoProject/Macao/GamePlaceForm.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Macao
+{
+    public partial class GamePlaceForm : Form
+    {
+        public List<PictureBox> Player1Cards {  get; set; } = new List<PictureBox>();
+
+        public List<PictureBox> Player2Cards { get; set; } = new List<PictureBox>();
+
+        public List<PictureBox> RemainingCards { get; set; } = new List<PictureBox>();
+
+        public List<PictureBox> MoveCards { get; set; } = new List<PictureBox>();
+
+        public GamePlaceForm()
+        {
+            InitializeComponent();
+        }
+
+        public void AddPlayer1Cards(PictureBox picture)
+        {
+            Player1Cards.Add(picture);
+        }
+
+        public void AddPlayer2Cards(PictureBox picture)
+        {
+            Player2Cards.Add(picture);
+        }
+
+        private void GamePlaceForm_Load(object sender, EventArgs e)
+        {
+            Deck deck = new Deck();
+            Game newGame = new Game();
+            Deck startDeck = newGame.CreateDeck();
+            newGame.StartGame(startDeck);
+
+            int startDeckSize = startDeck.Cards.Count;
+            int counterPlayerCard = 0;
+            Card startCard = new Card();
```

Comment:
extract in private method like SetStartCard SetPlayer1Cards and so on so that the code is more clear. Each private method should do 1 thing

you can add in the already existing methods like AddPlayer2Cards, but add all the code necessary to create the player 2 cards for example